lib/features/thread/presentation/styles/item_email_tile_styles.dart

```diff
@@ -27,7 +27,7 @@ class ItemEmailTileStyles {
 
   static EdgeInsetsGeometry getPaddingDividerWeb(BuildContext context, ResponsiveUtils responsiveUtils) {
     if (responsiveUtils.isDesktop(context)) {
-      return const EdgeInsetsDirectional.only(start: 120, top: 2);
+      return const EdgeInsetsDirectional.only(start: 180, top: 2);
```

Comment:
why we need to change it?

Reply:
The divider will be too long otherwise. There are more item in the front of the Avatar now, we need to adjust the divider.

Reply:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/89934827-3ba7-4874-8764-81f3dfa11337">

Divider still need to cover avatar

Reply:
It currently is. Idk what you mean